node/primitives/common/src/lib.rs

```diff
@@ -116,19 +116,37 @@ pub const MILLISECS_PER_BLOCK: Moment = 6000;
 /// Minute in Pangoro/Pangolin.
 pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 /// Hour in Pangoro/Pangolin.
+#[cfg(feature = "fast-runtime")]
```

Comment:
In order to make the test process easier?

Reply:
Yes.